cpp/arcticdb/pipeline/write_options.hpp

```diff
@@ -36,6 +37,7 @@ struct WriteOptions {
     bool ignore_sort_order = false;
     bool bucketize_dynamic = false;
     size_t max_num_buckets = 150;
+    bool compact_incomplete_dedup_rows = false;
```

Comment:
Not an option on ArcticDB, if people want to dedup then that should be via the processing pipeline, not as a surprise behaviour via a library setting